opensrp-reveal/src/test/java/org/smartregister/reveal/presenter/ListTaskPresenterTest.java

```diff
@@ -112,14 +137,172 @@ public void testOnLocationValidatedCallsStartFormWithCorrectArgumentsForMosquito
 
         PowerMockito.doNothing().when(listTaskPresenterSpy, "startForm", any(), any(), anyString());
```

Comment:
Remove this, the object being tested should not be disabled from executing any methods, only the dependencies should be disabled  or mocked.

Add a verify that the `listTaskView` was invoked to start the form
  ` listTaskView.startJsonForm(formJson);`

Reply:
Done.